public/models/aeon_record_mapper.rb

```diff
@@ -119,6 +149,8 @@ def system_information
             mappings['aeon_site_code'] = self.repo_settings[:aeon_site_code]
         end
 
+        mappings['Site'] = self.repo_settings[:site] if self.repo_settings.has_key?(:site)
```

Comment:
We used lower_case_with_underscore casing `(aeon_site_code)` for the Aeon Site mapping, in order to match the casing of the other mappings that are routed to fields in the Aeon Transaction. The other mappings in this `system_information` section are cased using PascalCase, because they have special meanings for the endpoint that we're using.

Would you prefer to keep the `Site` field as it is currently cased? We can add both as options, since the **OpenURL Field Values** column of the Open URL Mapping table can be pipe delimited.

Reply:
See my comment above about why I'm sending it as `Site`.

It seems that if I send form inputs with names that match column names in Aeon, then the values end up in the right places. Am I missing something? I feel like I must be because I don't understand why I would want to be dependent on an openurl mapping if I don't have to be.

Reply:
No, you're not missing anything, you're right. I assumed that all data had to pass through the OpenURLMapping table in order to get into Aeon, but any field names that already match Aeon Transaction fields will be imported automatically. There's no reason why this one needs its own special OpenURLMapping.

Since the Site field is a special field in Aeon, I'm inclined to take your `mappings['Site']` instead of the `mappings['aeon_site_code']` that was added last week. However, could you use the `self.repo_settings[:aeon_site_code]` repository setting for this change? We have sites that may already be using the `:aeon_site_code` term. Is Yale already using the `:aeon` term?

Reply:
That will be fine - Yale isn't in production yet so it will have no impact on them.

I'll push a commit shortly with this change.